src/platforms/ios/ioscontroller.mm

```diff
@@ -4,14 +4,14 @@
 
 #include "ioscontroller.h"
 #include "Mozilla_VPN-Swift.h"
-#include "controller.h"
 #include "ipaddress.h"
 #include "leakdetector.h"
 #include "logger.h"
 #include "models/device.h"
 #include "models/keys.h"
 #include "models/feature.h"
 #include "models/server.h"
```

Comment:
ios build wants: `#include "modules/vpn.h"` and `ModuleVPN::instance()->controller();` on L85